guides/controllers/request-and-response-objects.md

```diff
@@ -35,3 +35,10 @@ The response object is not usually used directly, but is built up during the exe
 | charset | The character set being used for the response. Default is "utf-8". |
 | headers | Headers used for the response. |
 
+## Multi params
+
+If you need get array of params from query:
+`?brand[]=brand1&brand[]=brand2&brand[]=brand3`
+
+try to fetch them:
+`params.fetch_all("brand[]")` => `{ "brand[]" => ["brand1", "brand2", "brand3"] }` 
```

Comment:
Please use triple back tricks here, this way is easier to read on mobile. (GitBook adds a nice overflow sidebar)

Like this

~~~
## Multi params

If you need get array of params from query:

```
?brand[]=brand1&brand[]=brand2&brand[]=brand3
```

Try to fetch them:

```
params.fetch_all("brand[]") => { "brand[]" => ["brand1", "brand2", "brand3"] } 
```
~~~